core/src/bls_sigverify/bls_sigverify_service.rs

```diff
@@ -0,0 +1,165 @@
+use {
+    crate::bls_sigverify::{bls_sigverifier::BLSSigVerifier, stats::BLSPreVerifyPacketStats},
+    agave_votor_messages::consensus_message::ConsensusMessage,
+    core::time::Duration,
+    crossbeam_channel::{Receiver, RecvTimeoutError, TrySendError},
+    solana_measure::measure::Measure,
+    solana_perf::{
+        deduper::{self, Deduper},
+        packet::PacketBatch,
+    },
+    solana_streamer::streamer::{self, StreamerError},
+    solana_time_utils as timing,
+    std::thread::{self, Builder, JoinHandle},
+    thiserror::Error,
+};
+
+#[derive(Error, Debug)]
+pub enum BLSSigVerifyServiceError {
+    #[error("try_send packet errror")]
```

Comment:
should we include the specific error? `{0}`

Reply:
TrySendError is big, so I'm returning the error kind as a string, does that work?